lib/git_worktree.rb

```diff
@@ -167,6 +167,19 @@ def file_list
     tree.walk(:preorder).collect { |root, entry| "#{root}#{entry[:name]}" }
   end
 
+  # Like "file_list", but doesn't return directories
```

Comment:
This isn't quite right because file_list doesn't return blobs.  I think this comment can be removed.